axelor-account/src/main/java/com/axelor/apps/account/service/move/MoveValidateServiceImpl.java

```diff
@@ -641,7 +642,12 @@ protected void checkInactiveJournal(Move move) throws AxelorException {
 
   protected void validateVatSystem(Move move) throws AxelorException {
     if (!CollectionUtils.isEmpty(move.getMoveLineList())) {
-      if (isConfiguredVatSystem(move) && isConfigurationIssueOnVatSystem(move)) {
+      if ((move.getJournal().getJournalType().getTechnicalTypeSelect()
```

Comment:
Need to add a check to now if journal type is filled, else we will get a NPE since the field is not required.